Modelica/Electrical/Analog/Examples/ResonanceCircuits.mo

```diff
@@ -6,7 +6,7 @@ model ResonanceCircuits
   parameter SI.Capacitance C=0.01 "Capacitance";
   parameter SI.Inductance L=0.01 "Inductance";
   final parameter SI.Frequency fRes=1/(2*pi*sqrt(L*C)) "Source frequency";
```

Comment:
```suggestion
  final parameter SI.Frequency fRes=1/(2*pi*sqrt(L*C)) "Resonance frequency";
```
I agree that the next line should say "Source to resonance frequency", but that doesn't make sense in itself if both of the frequencies are called 'Source frequency'.